File: 1406-subtract-the-product-and-sum-of-digits-of-an-integer/1406-subtract-the-product-and-sum-of-digits-of-an-integer.js
var subtractProductAndSum = function(n) {
    let digits = n.toString().split('').map(Number);
    
    let product = digits.reduce((a, d) => a * d, 1);
    
    let sum = digits.reduce((a, d) => a + d, 0);
    
    return product - sum;
};

File: 1406-subtract-the-product-and-sum-of-digits-of-an-integer/1406-subtract-the-product-and-sum-of-digits-of-an-integer.cs
public class Solution {
    public int SubtractProductAndSum(int n) {
    int product = 1;
    int sum = 0;

    foreach (char c in n.ToString())
    {
        int digit = c - '0'; 
        product =product*digit;
        sum =sum+ digit;
    }

    return product - sum;
    }
}
 

    

 
